contrib/kl_layer/README.md

```diff
@@ -0,0 +1,18 @@
+This folder contains the implementation of custom KL divergence layer.
+This is a custom implementation and not a built-in layer in any deep learning framework.
+It was developed specifically for [AD@L1 CMS paper](https://www.nature.com/articles/s42256-022-00441-3) and works spesifically for the input of size `(19, 3, 1)`.
```

Comment:
Does it really work `spesifically for the input of size (19, 3, 1)`? I see no hardcoded shapes in the implementation

Reply:
right. fixed now, thanks